management/sites/sites.php

```diff
@@ -0,0 +1,318 @@
+									class="wdn-button wdn-button-brand"
+									ng-click="addShiftCancelButtonHandler()">Cancel</button>
+								<input type="submit" 
+									value="Save" 
+									class="submit wdn-button wdn-button-triad"
+									ng-disabled="!form.$valid || savingShift === true">
+							</div>
+						</form>
+					</div>
+
+					<!-- Button for adding a shift -->
+					<button class="wdn-button margin-top-10" 
+						ng-click="addShiftButtonHandler()" 
+						ng-if="addShiftButtonClicked === false">Add Shift</button>
+				</div>
+
+
+				<!-- Appointment Times Section -->
+				<h4>Appointment Times:</h4>
+				<div>
+					<p>Appointment times represent a time slot in which a client can sign up for an appointment. Additionally, appointment times control the rules around the number of appointments that can be scheduled during the time slot.</p>
+					<p>Appointment times must follow these rules:</p>
+					<ul>
+						<li>Scheduled time must be within a volunteer shift</li>
+						<li>Scheduled time + approximate length in minutes must be within a volunteer shift</li>
+						<li>Two appointment times cannot have the same scheduled time</li>
+					</ul>
+
+					<!-- Displayed when there are no appointment times -->
+					<div ng-if="siteInformation.appointmentTimes == undefined || siteInformation.appointmentTimes.length <= 0">
+						<p class="clear-top">There are no appointment times</p>
+					</div>
+
+					<!-- Displayed when there are appointment times -->
+					<div ng-if="siteInformation.appointmentTimes != undefined && siteInformation.appointmentTimes.length > 0">
+						<table class="wdn_responsive_table">
+							<thead>
+								<tr>
+									<th id="appointmentTimeScheduledTimeHeader">Scheduled Time</th>
+									<th id="appointmentTimeMinimumNumberOfAppointmentsHeader">
+										Min Appts										
+										<a class="tooltip pointer" title="This is the minimum number of appointments for this time slot, meaning at least this many appointments will be allowed to be scheduled even if there are not enough preparers signed up during this time. Default is 0.">
+											<span class="wdn-icon-info" aria-hidden="true"></span>
+										</a>
+									</th>
+									<th id="appointmentTimeMaximumNumberOfAppointmentsHeader">
+										Max Appts
+										<a class="tooltip pointer" title="This is the maximum number of appointments for this time slot, meaning even if there are more preparers than this number, this is the number of appointments that will be allowed to be scheduled. Default is N/A.">
+											<span class="wdn-icon-info" aria-hidden="true"></span>
+										</a>
+									</th>
+									<th id="appointmentTimePercentageAppointmentsHeader">
+										Percentage Appts
+										<a class="tooltip pointer" title="This specifies the percentage of the preparers that are allotted for appointments, that is: number of appointments allowed to be scheduled = number of preparers * percentage appointments. Default is 100%.">
```

Comment:
While it is all correct information, I'm not sure that this description will be clear to the users. I believe the purpose of the percentage appts value is to only allow a certain percentage of the appointments to be scheduled online, thus reserving the other appointments for walk-ins. I wonder if we should hint that the usage/purpose of this value is to save some the time of some preparers for walk-ins. I'm thinking something along these lines...

"This specifies the percentage of the preparers that are allotted for scheduled (as opposed to walk-in) appointments. That is: number of appointments allowed to be scheduled online = number of preparers * percentage appointments. Default is 100%."

Thoughts?

Reply:
Good idea, I'll make that change. On this note though, in my talk with Dr. Moody and Katie, this year, they want appointment times to be on the hour (i.e 1pm, 2pm, 3pm), and then have 2 appointments allowed to be scheduled per preparer... I was planning on using the `percentageAppointments` functionality to achieve this result. Do you have any thoughts on this?

Reply:
Interesting. So they want to intentionally overbook? In terms of using `percentageAppointments` to achieve this, are you thinking we would allow the value to be set above 100%? Do we just want to increase the upper limit to something like 300%?

Reply:
Yah they do want to intentionally overbook, but they made it sound like appointments weren't taking a full hour in most cases, but they didn't want to shorten appointment times to 30 minutes because on the hour gave them more flexibility. So my thought was to set `percentageAppointments` to 200% on a lot of the appointment times in order to achieve this. 

So my initial thought was just to remove the upper limit constraint on the `percentageAppointment` field, but I think an upper limit like 300% is more appropriate. You think this is a "good" way to achieve the result we're after? At least initially to see if it is something that even works out for them?